### PR TITLE
LRDOCS-3535 developer mode 7.0 docs

### PR DESCRIPTION
**Pull request by mwilliams2014**
cc @blzaugg

### CODE REVIEW

File: develop/tutorials/articles/260-themes-and-layout-templates/07-using-developer-mode-with-themes.markdown [hunk 1]
```diff
@@ -52,41 +68,20 @@ file has the property setting:
...
+to learn how to manually set Developer Mode in your app server. 
 
 ## Setting Developer Mode for Your Server Using portal-developer.properties [](id=setting-developer-mode-for-your-server-using-portal-developer-properties)
 
-To set Developer Mode manually, you must add the `portal-developer.properties`
+To set Developer Mode manually, You must add the `portal-developer.properties` 
 file to your application server's launch configuration. Since each application 
 server has a different configuration file or UI to specify system properties, 
 you must follow your application server's specific method for adding the 
 `portal-developer.properties` file to the server's launch configuration.
```
Comment (blzaugg) [inline develop/tutorials/articles/260-themes-and-layout-templates/07-using-developer-mode-with-themes.markdown]: @mwilliams2014, I still have an issue with this section. I don't think this is nessesary when using `portal-ext.properties`.

Comment (mwilliams2014) [inline develop/tutorials/articles/260-themes-and-layout-templates/07-using-developer-mode-with-themes.markdown]: Ok. If you know that this is not needed, I'll remove this section and just say that you just have to add the include-and-override or copy the portal-developer properties over. thx! I'll close and resend.

Comment (mwilliams2014) [inline develop/tutorials/articles/260-themes-and-layout-templates/07-using-developer-mode-with-themes.markdown]: Before, I update that section, are you sure that `portal-ext.properties` is all you need, regardless of the app server @blzaugg?

**Comment by mwilliams2014** [comment]
@blzaugg updated

File: develop/tutorials/articles/260-themes-and-layout-templates/07-using-developer-mode-with-themes.markdown (hunk 1 above)
Comment (blzaugg) [inline develop/tutorials/articles/260-themes-and-layout-templates/07-using-developer-mode-with-themes.markdown]: I don't know that. It's my assumption. I assume Portal is explicitly looking for `portal-ext.properties` and it doesn't depend on the app server.

**Comment by sez11a** [comment]
View total diff: https://github.com/sez11a/liferay-docs/compare/0487f7fbf1...2aa26785b2

**Pull request closed without merging**